packages/cursorless-everywhere-talon/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "@cursorless/cursorless-everywhere-talon",
+  "version": "1.0.0",
+  "description": "cursorless in talon js",
+  "main": "./out/index.js",
+  "license": "MIT",
+  "type": "module",
+  "private": true,
+  "scripts": {
+    "esbuild:dev": "pnpm run esbuild:base src/mainDevelopment.ts",
+    "esbuild:prod": "pnpm run esbuild:base --minify src/mainProduction.ts",
+    "esbuild:watch": "pnpm run esbuild:base --watch src/mainDevelopment.ts",
+    "esbuild:meta": "pnpm run esbuild:base --metafile=meta.json src/mainDevelopment.ts",
```

Comment:
should we call this `esbuild:profile`? At first read it's tough to know what `meta` is supposed to mean here. I know you're just borrowing esbuild terminology but I think profiling is actually what you use this for right?

Reply:
Sounds good